### PR TITLE
888121 - MAUI - Prepare the UG documentation for row header support

### PR DESCRIPTION
**Pull request by AbhishekSF4201**
Added show row header

### CODE REVIEW

File: MAUI/DataGrid/Column-types.md [hunk 1]
```diff
@@ -908,6 +908,30 @@ dataGrid.Columns.Add(numericColumn);
 {% endhighlight %}
 {% endtabs %}
 
+### Show row header
+
+The `SfDataGrid` allows you to add the content inside the row header using the RowHeaderTemplate. It allows you to load any view inside the row header.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/Column-types.md:913]: The row header is a type of column that is placed as the first cell of each row and remains frozen. To enable the row header, set `SfDataGrid.ShowRowHeader` to `true`.

Additionally, the `SfDataGrid` allows you to customize the row header width using the `SfDataGrid.RowHeaderWidth` property. The default value is 25.

File: MAUI/DataGrid/Column-types.md [hunk 2]
```diff
@@ -908,6 +908,30 @@ dataGrid.Columns.Add(numericColumn);
 {% endhighlight %}
 {% endtabs %}
 
+### Show row header
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/Column-types.md:911]: Row header

File: MAUI/DataGrid/conditional-styling.md [hunk 3]
```diff
@@ -627,3 +627,52 @@ public class ColorConverter : IValueConverter
 <img alt="Conditional styling for unbound row using converter" src="Images/conditional-styling/maui-datagrid-conditional-unbound-row.png" width="404"/>
 
 
+## Show row header
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/conditional-styling.md:630]: Row header

File: MAUI/DataGrid/styling.md [hunk 4]
```diff
@@ -325,6 +325,21 @@ The unbound row can be customized by the writing style for [DataGridUnboundRowVi
 
 <img alt="Unbound row styling in .NET MAUI DataGrid" src="Images\styling\maui-datagrid-unbound-row-styling.png" width="404"/>
 
+### Styling show row header
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/styling.md:328]: Styling row header

File: MAUI/DataGrid/styling.md [hunk 5]
```diff
@@ -325,6 +325,21 @@ The unbound row can be customized by the writing style for [DataGridUnboundRowVi
...
+The row header can be customized by the writing style for DataGridRowHeaderCell Target Type.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/styling.md:329]: The row header can be customized by the writing style for `DataGridRowHeaderCell` Target Type

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

**Comment by AbhishekSF4201** [comment]
This implementation has done in another pull request.
https://github.com/syncfusion-content/maui-docs/pull/2300/

**Pull request closed without merging**